proto/cometbft/abci/v2/types.proto

```diff
@@ -39,13 +41,20 @@ message Request {
   reserved 4;
 }
 
+
+// Handling v1 as v2: ACCEPT
+//     SET abci_version:v2 TO "<NOT PROVIDED>"
```

Comment:
```suggestion
//     follow normal protobuf rules
```

Reply:
(from discussionwith @mzabaluev): Consider removing this, as this is considered as the default (so only document the "REJECT"s as counter-intuitive